lib/delayed_job_groups_plugin.rb

```diff
@@ -11,6 +11,15 @@
 require 'delayed/job_groups/yaml_loader'
 require 'delayed/job_groups/version'
 
-Delayed::Backend::ActiveRecord::Job.send(:include, Delayed::JobGroups::JobExtensions)
+if defined?(Delayed::Backend::ActiveRecord)
+  if defined?(Rails::Railtie)
+    # Postpone initialization to railtie for correct order
+    require 'delayed/job_groups/backend/active_record/railtie'
+  else
+    # Do the same as in the railtie
+    Delayed::Backend::ActiveRecord::Job.send(:include, Delayed::JobGroups::JobExtensions)
```

Comment:
While you're in here, can drop the `send` since this gem only support Ruby 2.5+ now?

```suggestion
    Delayed::Backend::ActiveRecord::Job.include(Delayed::JobGroups::JobExtensions)
```